cmd/metacache.go

```diff
@@ -170,8 +170,8 @@ func (m *metacache) canBeReplacedBy(other *metacache) bool {
 	if m.status == scanStateStarted && time.Since(m.lastUpdate) < metacacheMaxRunningAge {
 		return false
 	}
-	// Keep it around a bit longer.
-	if time.Since(m.lastHandout) < time.Hour || time.Since(m.lastUpdate) < metacacheMaxRunningAge {
+
+	if time.Since(m.lastUpdate) < metacacheMaxRunningAge {
```

Comment:
No, someone could still be listing from it. When using continuation tokens old caches needs to be available for a while.

Reply:
This has the potential to cancel active listings?;

Reply:
Yes, listings with continuation tokens will reference the cached list directly. `m.lastHandout` keeps track of when the ID was last seen.

Reply:
The problem I have is until 1hr, as many requests are made if the endCycle <= OldestCycle we get caches are generated for all requests.